servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/AbstractQuarkusEvents.java

```diff
@@ -507,9 +508,17 @@ private Branch mainBranch() throws NessieNotFoundException {
   }
 
   private String getMetrics() {
+    int managementPort = Integer.getInteger("quarkus.management.port");
+    URI managementBaseUri;
+    try {
+      managementBaseUri =
+          new URI("http", null, clientUri.getHost(), managementPort, "/", null, null);
```

Comment:
Why not inject it like `@NessieClientUri`?

Reply:
It's rather rare - so :shrug:

Reply:
fair enough